Client.xcodeproj/project.xcworkspace/xcshareddata/Client.xcscmblueprint

```diff
@@ -23,6 +25,11 @@
   "DVTSourceControlWorkspaceBlueprintRelativePathToProjectKey" : "Client.xcodeproj",
   "DVTSourceControlWorkspaceBlueprintRemoteRepositoriesKey" : [
     {
+      "DVTSourceControlWorkspaceBlueprintRemoteRepositoryURLKey" : "https:\/\/github.com\/fluffyemily\/firefox-menu-experiments.git",
+      "DVTSourceControlWorkspaceBlueprintRemoteRepositorySystemKey" : "com.apple.dt.Xcode.sourcecontrol.Git",
+      "DVTSourceControlWorkspaceBlueprintRemoteRepositoryIdentifierKey" : "17D28135537A0534CA3953E850B2215BF7F6515F"
+    },
+    {
       "DVTSourceControlWorkspaceBlueprintRemoteRepositoryURLKey" : "github.com:mozilla\/firefox-ios.git",
       "DVTSourceControlWorkspaceBlueprintRemoteRepositorySystemKey" : "com.apple.dt.Xcode.sourcecontrol.Git",
```

Comment:
that'll probably be through a rebase against master that I didn't push from
the parent branch. Let me fix that.

On 23 March 2016 at 18:04, Stephan Leroux <notifications@github.com> wrote:

> In Client.xcodeproj/project.xcworkspace/xcshareddata/Client.xcscmblueprint
> <https://github.com/mozilla/firefox-ios/pull/1620#discussion_r57207949>:
>
> > @@ -23,6 +25,11 @@
> >    "DVTSourceControlWorkspaceBlueprintRelativePathToProjectKey" : "Client.xcodeproj",
> >    "DVTSourceControlWorkspaceBlueprintRemoteRepositoriesKey" : [
> >      {
> > +      "DVTSourceControlWorkspaceBlueprintRemoteRepositoryURLKey" : "https:\/\/github.com\/fluffyemily\/firefox-menu-experiments.git",
> > +      "DVTSourceControlWorkspaceBlueprintRemoteRepositorySystemKey" : "com.apple.dt.Xcode.sourcecontrol.Git",
> > +      "DVTSourceControlWorkspaceBlueprintRemoteRepositoryIdentifierKey" : "17D28135537A0534CA3953E850B2215BF7F6515F"
> > +    },
> > +    {
> >        "DVTSourceControlWorkspaceBlueprintRemoteRepositoryURLKey" : "github.com:mozilla\/firefox-ios.git",
> >        "DVTSourceControlWorkspaceBlueprintRemoteRepositorySystemKey" : "com.apple.dt.Xcode.sourcecontrol.Git",
>
> Not sure where these changes came from
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/firefox-ios/pull/1620/files/d472ac87098126fd4eebbcc56e73b5b91eeb762d#r57207949>
>